frontend/src/features/workspace/components/AdminSwitchEnvMessage.tsx

```diff
@@ -63,7 +52,7 @@ export const AdminSwitchEnvMessage = (): JSX.Element => {
             as="u"
             variant="link"
             display="inline"
-            onClick={onOpen}
+            onClick={handleClick}
```

Comment:
Just checking: is `handleClick` as a callback necessary? Could we just:
```suggestion
            onClick={adminSwitchEnvMutation.mutate}
```

Reply:
not necessary, but just wrapped it to match the naming conventions we usually use!